.travis.yml

```diff
@@ -117,3 +117,19 @@ after_success:
     --exclude="debug_helpers" --exclude="example" --exclude="external_example"
     --exclude-pattern=".*CMakeFiles.*" --exclude-pattern=".*_test.*";
   fi
+
+before_deploy:
+- export RELEASE_PKG_FILE=$(ls *.deb)
+- echo "Deploying $RELEASE_PKG_FILE to GitHub releases"
+
+deploy:
+  provider: releases
+  skip_cleanup: true
+  api_key:
+    secure: hBX3pFWNZiDbz4yKnOjhLg3QS9Ubn1XePxSeIt2Btq5GzbomOPDCgpIFijBppliwj9oKc302EMnZSg2QWeAzFKn9UnmIflJ0E4iymYgwWdTJv+bSnYALJEmO8F6gF9FgRlPk8FCtZiECoTsa75w5TrEZKZpFpmzVYRiDu0eo6sEjW7UJPC0A2KSTXLrBCHSIZy/iasbGmuur4brG7NO0QdMOvDXvhsYfkXDRJFMTtTHvLiKJcqiunPfqARzf1H4x4iczRYscKu5Vn8Kmw3NANGkcIDvEj4ooih831EXxACRZw0VgycgNHOKRXKC9pZ4hLQMon+jxpQX+X8k/K5161oEkF/gCVKyFb31Pk/4Uwe81p1GJY2lAC7MDUxA98RKXhdvVYF2Cp44+IbF0YVoWRUtVAhknXRQ3Weg25kyVSu83q2nN2nZq2qGTnpNIbdN56s/F+uaFtipGEh+vmiv8rNUz+Z5MFrY2FQaSvBTFw9K4tNs9uc+VQd1bE7X5wh0yywEqUEw2nzqTB2xR+OubygUASbk2GLNdc254P0lrzCHbNM62Y7sRX06CM7hPlwhELEkVtUXZWJ0KuhQyLvRh3aPJ3Jj30EswTt/FGT1gzSP1FjjHBRZCK4P2D2rwJ5TMn2JrZKfPxmEd3kVmn6h80+gBbKgonGmZspd2SvPEI5g=
```

Comment:
Probably a stupid question, but if this key is secure, why is it public here?

Reply:
You only see the encrypted string. It's a common way with travis.
https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml